ME-MYSELF-I/Weekly-meetings.md

```diff
@@ -1,8 +1,38 @@
 # Weekly Meetings
 
+* [22nd September 2017](#date-22nd-september-2017)
 * [14th September 2017](#date-14th-september-2017)
 
 
+### Date: 22nd September 2017
+
+#### Who did you help this week?
+
+I helped (and am in the process of helping) the ReproZip team streamline their Dockerfile and develop methods to minimize Docker containers (see [ViDA-NYU/reprozip#274](https://github.com/ViDA-NYU/reprozip/issues/274)). I also helped Dorota [@djarecka](https://github.com/djarecka) debug a workflow that uses Common Workflow Language.
+
+#### Who helped you this week?
+
+[Satra](https://github.com/satra) helped me by providing guidelines for regression testing and by linking me to existing networks for brain image segmentation. Dorota [@djarecka](https://github.com/djarecka) helped me by introducing me to Common Workflow Language syntax.
+
+#### What did you achieve?
+
+* Created [a matrix of Docker images](https://hub.docker.com/r/kaczmarj/regtests/tags/) to test with [ReproNim/simple_workflow](https://github.com/ReproNim/simple_workflow).
+  * Tested simple_workflow in an FSL 5.0.10 Docker container, and minified that container for the workflow.
+* Downloaded BrainBox NIFTI files to OpenMind. These files will be used to train/test a net for nobrainer.
+* Neurodocker bug fixes.
+
+#### What did you struggle with?
+
+* Training a neural net on the BrainBox NIFTI data. I had difficulty organizing the data in the form required for each net.
```

Comment:
do all the 3d models use the same formulation?

Reply:
i was getting hung up because oneof the models were trains on brats data, and I did not know the structure of that dataset. now that i am looking again, niftynet should be quick to run because the paths to train/test data can be customized. i will try running niftynet tomorrow (sunday).